easybuild/easyconfigs/p/Python/Python-2.7.12-GCCcore-4.9.3-bare.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Python'
+version = '2.7.12'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.14.1'),
+    #   ('OpenSSL', '1.0.1k'),  # OS dependency should be preferred if the os version is more recent then this version, it's
+    #   nice to have an up to date openssl for security reasons
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev')]
+
+# We hide this by default since users should not use it in production
+#hidden = True
```

Comment:
@ocaisa can you clarify why this should not be used in production in the comment?
If it's clear, I guess it makes sense to leave this in.

Also, please use `--pr-commit-msg` with `--update-pr` to provide meaningful commit messages; I'll look into enforcing that (only `--new-pr` defaults to somewhat decent auto-derived commit messages)